ckanext/canada/tables/nap5.yaml

```diff
@@ -0,0 +1,1333 @@
+dataset_type: nap5
+
+title: 5th National Action Plan on Open Government Tracker
+notes: |
+  Access, upload and modify the National Action Plan on Open Government Tracker for your organization
+
+template_version: 3
+template_updated: 2022-06-08
+
+portal_type: info
+collection: pd
+
+resources:
+- title: 5th National Action Plan on Open Government Tracker (draft)
+  resource_name: nap5
+  create_form: true
+  edit_form: true
+
+  fields:
+  - datastore_id: reporting_period
+    label:
+      en: Reporting Period
+      fr: Période de déclaration
+    description:
+      en: . (TBD?)
+      fr: . (TBD?)
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    datastore_type: text
+    choices:
+      2022-03: 2022-03
+      2022-06: 2022-06
+      2022-09: 2022-09
+      2022-12: 2022-12
+      2023-03: 2023-03
+      2023-06: 2023-06
+      2023-09: 2023-09
+      2023-12: 2023-12
+      2024-03: 2024-03
+      2024-06: 2024-06
+      2024-09: 2024-09
+      2024-12: 2024-12
+
+  - datastore_id: commitments
+    label:
+      en: Commitments
+      fr: Engagements
+    description:
+      en: The 5 commitments in the 2022-2024 National Action Plan on Open Government
+      fr: Les 5 engagements dans le Plan d’action national du Canada pour un gouvernement ouvert de 2022-2024
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    datastore_type: text
+    excel_full_text_choices: true
```

Comment:
you could add an option like
```yaml
form_choices_include_codes: true
```
that will add the codes to the select text similar to how `excel_full_text_choices` adds the text to the options in excel